### PR TITLE
add permission function to check for authorized key

### PR DESCRIPTION
**Pull request by cleder**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Customize the test machine
 machine:
     python:
-        version: 3.4.4
+        version: 3.5.1
```
Comment (komar) [inline circle.yml]: why not 3.5.2?

**Comment by komar** [comment]
Looks 👌

File: circle.yml (hunk 1 above)
Comment (cleder) [inline circle.yml]: takes ages as it is compiled from source every time - also seems to fail intermittently due to install errors of 3.5.2

Comment (cleder) [inline circle.yml]: also 3.5.2 is tested on https://travis-ci.org/PrimarySite/drf-deny-allow-pc/builds/158708973

File: setup.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
-from setuptools import setup, find_packages
-import sys
 import os
+import sys
+
+from setuptools import find_packages
+from setuptools import setup
 
 version = '0.2'
```
Comment (amusikal) [inline setup.py]: Need to bump version?

**Pull request merged**